api/constants/constants.go

```diff
@@ -17,6 +17,37 @@ var (
 	EulaMessage string
 )
 
+const (
+	FullGroupName = "symphony"
+
+	// system annotations, reserved and should not be modified by client.
+	AzureCorrelationIdKey        = "management.azure.com/correlationId"
+	AzureEdgeLocationKey         = "management.azure.com/customLocation"
+	AzureOperationIdKey          = "management.azure.com/operationId"
+	AzureNameIdKey               = "management.azure.com/azureName"
+	AzureResourceIdKey           = "management.azure.com/resourceId"
+	AzureSystemDataKey           = "management.azure.com/systemData"
+	AzureTenantIdKey             = "management.azure.com/tenantId" // Not used
+	RunningAzureCorrelationIdKey = "management.azure.com/runningCorrelationId"
+	SummaryJobIdKey              = "SummaryJobIdKey"
```

Comment:
If this deem to be a system annotation, should it be put under management.azure.com/ as well?